examples/deployments/aws-terraform/chroma.tf

```diff
@@ -102,26 +102,41 @@ resource "aws_instance" "chroma_instance" {
 }
 
 
-resource "aws_ebs_volume" "chroma-volume" {
+resource "aws_ebs_volume" "chroma_volume" {
+  count = var.prevent_chroma_data_volume_delete ? 0 : 1
+
   availability_zone = aws_instance.chroma_instance.availability_zone
   size              = var.chroma_data_volume_size
 
   tags = {
     Name = "chroma"
   }
+}
+
+resource "aws_ebs_volume" "chroma_volume_prevent_destroy" {
+  count = var.prevent_chroma_data_volume_delete ? 1 : 0
+
+  availability_zone = aws_instance.chroma_instance.availability_zone
+  size              = var.chroma_data_volume_size
 
-  lifecycle {
-    prevent_destroy = var.prevent_chroma_data_volume_delete # size in GBs
+  tags = {
+    Name = "chroma"
   }
+
+#  lifecycle {
+#    prevent_destroy = true
```

Comment:
Nit: Should we set this to `true` and instruct users to set it to `false` if they want to delete their volume? That seems in line with what the documentation says atm (though I'm fine with it either way)